quickstart/overview.markdown

```diff
@@ -3,24 +3,24 @@ title: Overview of the ALICE O<sup>2</sup>
 layout: main
 ---
 
-This is an overview of the O<sup>2</sup> system. It is pretty rough but it should give you an idea of the general architecture. Please refer to other documents such as the [TDR](https://cds.cern.ch/record/2011297).
+This is an overview of the O<sup>2</sup> system. It is pretty rough, but it should give you an idea of the general architecture. Please refer to other documents such as the [TDR](https://cds.cern.ch/record/2011297).
 
 ## Requirements
 
-1. After LS2 the ALICE experiment will have gone through a massive upgrade to handle minimum bias PbPb interactions at 50 kHz what results in a 100 times more data than in 2010.
+1. After LS2 the ALICE experiment will have gone through a massive upgrade to handle minimum bias PbPb interactions at 50 kHz. This will results in a 100 times more data than in Run 2.
```

Comment:
I checked with Pierre and it is compared to the beginning of Run 1. We should probably also say that it is the event rate rather than "data".
-> "This will result in a 100 times more events than in Run 1 (2010)."